CONTRIBUTING.md

```diff
@@ -4,3 +4,4 @@ Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org
 
 1. [Issues marked as `good-first-bug`](https://github.com/mozilla/bugbug/labels/good-first-bug) are self-contained enough that a contributor should be able to work on them.
 2. Issues are considered not assigned, until there is a PR linked to them. Feel free to work on any unassigned issue, you don't need to ask first.
+3. If you have any problem, it could be already asnwerd in [Discussions](https://github.com/mozilla/bugbug/discussions), if not, feel free to start a new discussion in the [Q&A](https://github.com/mozilla/bugbug/discussions/categories/q-a) category.
```

Comment:
```suggestion
3. If you have any problem, it could be already answered in [Discussions](https://github.com/mozilla/bugbug/discussions), if not, feel free to start a new discussion in the [Q&A](https://github.com/mozilla/bugbug/discussions/categories/q-a) category.
```
Please fix the typo

Reply:
@srishtig2412 you marked this as resolved while the typo is still not fixes!!

Reply:
changed that..